src/open_inwoner/templates/pages/cases/submissions.html

```diff
@@ -0,0 +1,40 @@
+{% extends 'master.html' %}
+{% load i18n button_tags anchor_menu_tags card_tags grid_tags icon_tags link_tags list_tags pagination_tags utils %}
+
+{% block sidebar_content %}
+    {% anchor_menu anchors=anchors desktop=True %}
+{% endblock sidebar_content %}
+
+{% block content %}
+    <h1 class="h1" id="title">{% trans "Mijn aanvragen" %}</h1>
+
+    <div class="cases">
+        <h2 class="h2" id="submissions">{{ page_title }} ({{ submissions|length }})</h2>
+
+         <table class="table">
+            <thead class="table__heading">
+                <tr>
+                    <td class="table__item">{% trans "Formulier" %}</td>
+                    <td class="table__item">{% trans "Laatste wijziging" %}</td>
+                    <td class="table__item">{% trans "Einde geldigheid" %}</td>
+                    <td class="table__item">&nbsp;</td>
+                </tr>
+            </thead>
+            <tbody>
+                {% for submission in submissions %}
+                    <tr>
+                        <th class="table__header">{% link href=submission.vervolg_link text=submission.naam target="_blank"  %}</th>
```

Comment:
`target` is not a variable of the template tag. Link component uses by default `target='_blank'` if href starts with 'http'

Reply:
> Link component uses by default target='_blank' if href starts with 'http'

Ugh.. that seems like a bad assumption or generalisation, but let's not think about it for now and stash that on the template-tag issue pile. I'll remove the variable for now.